docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 [id="planning_development_work"]
 = Planning your development work
 
-When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow before you launch the application.
 
-{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the guided work item type hierarchy in the planner.
 
-This example walks you through the Scenario Driven Development planning process to plan your work.
+This example walks you through the Agile planning process to plan your work.
```

Comment:
"walks you through" seems a little informal for tech communication to me. I'd suggest something like "This is an example of planning your work using the Agile planning process".

Reply:
+1 Will change